x-pack/plugins/monitoring_collection/server/lib/prometheus_exporter.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AggregationTemporality, MetricReader } from '@opentelemetry/sdk-metrics-base';
+import {
+  PrometheusExporter as OpenTelemetryPrometheusExporter,
+  ExporterConfig,
+  PrometheusSerializer,
+} from '@opentelemetry/exporter-prometheus';
+import { KibanaResponseFactory } from '@kbn/core/server';
+
+export class PrometheusExporter extends MetricReader {
+  private readonly _prefix?: string;
+  private readonly _appendTimestamp: boolean;
+  private _serializer: PrometheusSerializer;
```

Comment:
I think this can be just `_serializer` - pretty sure the underscore is just copy-pasta from the otel js repo during my PoC work.

Reply:
you mean without the type? I can, but then TS will understand that this an `any`. Better to define the type as we're currently doing

Reply:
without the underscore I mean :)

Reply:
For some reason my brain didn't read the *underscore*  from your original comment lol. Yeah, I'll remove the underscore from these properties.

Reply:
It doesn't hurt to leave them, but there are more examples without _ than with in the codebase.